clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -96,6 +96,15 @@ def err_drv_cuda_host_arch : Error<
 def err_drv_no_sycl_libspirv : Error<
   "cannot find '%0'; provide path to libspirv library via '-fsycl-libspirv-path', or "
   "pass '-fno-sycl-libspirv' to build without linking with libspirv">;
+def warn_flag_no_sycl_libspirv
+    : Warning<"no-sycl-libspirv option is not meant for regular application "
```

Comment:
```suggestion
    : Warning<"`-fno-sycl-libspirv` is not meant for regular application "
```

Reply:
Thanks for pointing this out. I think warn_drv_unsupported_option_for_target actually is appropriate so I have used it here.